### DISCUSSION TITLE
Can't install VirtualBox

### DISCUSSION THREAD

**Discussion by sauntor** [elementary/triage#272] [Q&A] [answered]
### What Happened?

The following is the installation log:
[install.log](https://github.com/elementary/triage/files/12017958/install.log)

It gets many errors, if you leave this error along, then `VirtualBox` (the GUI)  can not run any vms

### Steps to Reproduce

As `What Happened?` says

### Expected Behavior

The VMs  created by VirtualBox can run normally

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Hardware Info

nvidia-driver-535

**Comment by lenemter** [comment, marked as answer]
You need to install kernel headers:

```
sudo apt install linux-headers-generic-hwe-22.04
```

and then start the virtualbox module:

```
sudo modprobe vboxdrv
```

**Reply by sauntor** [reply]
I installed `linux-headers-generic-hwe-22.04`, it works after reboot! Great! Thanks!

**Reply by sauntor** [reply]
I think ubuntu has a bad package naming rule, it's not as clearly as `openSUSE`, on which I can just guess the name of the package.
